.github/workflows/super-linter.yml

```
@@ -30,10 +30,12 @@ jobs:
           PROTOBUF_CONFIG_FILE: .protolint.yml
           VALIDATE_NATURAL_LANGUAGE: false
           VALIDATE_OPENAPI: false
+          VALIDATE_JAVASCRIPT_STANDARD: false
           VALIDATE_JSCPD: false
           VALIDATE_GO: false
           VALIDATE_GO_MODULES: false
           PYTHON_PYLINT_CONFIG_FILE: .pylintrc
           DEFAULT_BRANCH: "main"
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           FILTER_REGEX_EXCLUDE: .*.jsonnet
+          
```

yamllint

[error] trailing spaces

(trailing-spaces)